settings.gradle.kts

```diff
@@ -0,0 +1,43 @@
+pluginManagement {
+    includeBuild("build-logic")
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+plugins {
+    id("com.gradle.develocity") version "3.18.2"
+}
+
+develocity {
+    server = "https://develocity.sonar.build"
+    buildScan {
+        tag(if (System.getenv("CI").isNullOrEmpty()) "local" else "CI")
+        tag(System.getProperty("os.name"))
+        if (System.getenv("CIRRUS_BRANCH") == "master") {
+            tag("master")
+        }
+        if (System.getenv("CIRRUS_PR")?.isBlank() == false) {
+            tag("PR")
```

Comment:
Nice improvement! WDYT about adding the PR name / branch name too?

Reply:
Good suggestion! I just found that for such values Gradle offers "custom values" and not tags: https://docs.gradle.com/develocity/gradle-plugin/current/#adding_custom_values. I'll add one for branch name